pristine-zen-garden.html

```diff
@@ -0,0 +1,111 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
```

Comment:
Don't need to indent head/body as they are the only child elements of HTML.  Saves an indent. 
Throw me a mention on the commit